### PR TITLE
Create programs

### PR DESCRIPTION
**Pull request by andrewjcasal**
Programs pages - Finished
Setup pages - All, but the "Admins" page

### CODE REVIEW

File: src/Pages/CreateProgramDocuments.js [hunk 1]
```diff
@@ -24,67 +28,83 @@ const TableData = [{
...
-              <Col md={12}>
-                <input type="text" placeholder="Document Name" />
-              </Col>
-            </Row>
-            <div className="file-upload">
-                <span>Upload</span>
-                <input id="uploadBtn" type="file" className="upload" />
+class CreateProgramDocuments extends React.Component {
+  constructor() {
+    super()
```
Comment (mKleinCreative) [inline src/Pages/CreateProgramDocuments.js]: Don't need a super here.. can just do 
state = {
 files: []
}

File: src/Components/FooterButtons.js [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
-      <button className="save-button">SAVE</button>
-    }
-    {props.cancel &&
-      <button className="cancel-button">CANCEL</button>
+class FooterButtons extends React.Component {
+  render() {
+    let options = "";
+    let footerOptions = "";
+
+    if (this.props.save == "black") {
```
Comment (mKleinCreative) [inline src/Components/FooterButtons.js]: You're still using props here, are you going to call super(props) ? or is this valid.

File: src/Components/UI/Container/Container.js [hunk 3]
```diff
@@ -6,16 +6,36 @@ import Navbar from '../../../containers/NavBar/NavBar';
...
-      </div>
-    </div>
-  </div>
-);
+class Container extends React.Component {
+
+  render() {
+    let layout = null;
+
+    if (this.props.left) {
```
Comment (mKleinCreative) [inline src/Components/UI/Container/Container.js]: same here.

File: src/Pages/CreateProgramDocuments.js [hunk 4]
```diff
@@ -24,67 +28,83 @@ const TableData = [{
 }];
 const TableColumns = [
```
Comment (mKleinCreative) [inline src/Pages/CreateProgramDocuments.js]: We gotta figure out a way to streamline the process of making these table cells -.-

File: src/Pages/LegalRepresentative.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        <div>
+          <Row>
+            <Col md={12}><input type="text" placeholder="Name" /></Col>
+          </Row>
+          <Row>
+            <Col md={12}><input type="text" placeholder="Address" /></Col>
+          </Row>
+          <Row>
+            <Col md={4}><input type="text" placeholder="City" /></Col>
+            <Col md={4}><Dropdown title="State" options={["male","female"]}/></Col>
```
Comment (mKleinCreative) [inline src/Pages/LegalRepresentative.js]: State is male or female?

File: src/Components/FooterButtons.js (hunk 2 above)
Comment (andrewjcasal) [reply src/Components/FooterButtons.js]: I have no idea what the right convention for this is. This works, but if you've seen that the convention is to put in super, I'll do that.

**Pull request merged**